src/models/api.model.ts

```diff
@@ -1,13 +1,13 @@
 export namespace apiModel {
```

Comment:
namespace'ów raczej nie powinno się używać bo to jest przestarzała praktyka. Dodatkowo interfejsy nazywaj z wielkiej litery